client/lib/signup/hint-data.js

```diff
@@ -1,673 +1,677 @@
+	Podcast: i18n.translate( 'Podcast' ),
+	Poems: i18n.translate( 'Poems' ),
+	Poetry: i18n.translate( 'Poetry' ),
+	Pokémon: i18n.translate( 'Pokémon' ),
+	'Polical activisim': i18n.translate( 'Polical activisim' ),
+	'Policial candidate': i18n.translate( 'Policial candidate' ),
+	'Political analysis': i18n.translate( 'Political analysis' ),
+	Politics: i18n.translate( 'Politics' ),
+	Pollution: i18n.translate( 'Pollution' ),
+	Portfolio: i18n.translate( 'Portfolio' ),
+	'Portrait artist': i18n.translate( 'Portrait artist' ),
+	Pregnancy: i18n.translate( 'Pregnancy' ),
+	Presentations: i18n.translate( 'Presentations' ),
+	'Print design': i18n.translate( 'Print design' ),
+	Printing: i18n.translate( 'Printing' ),
+	'Private consulting': i18n.translate( 'Private consulting' ),
+	Product: i18n.translate( 'Product' ),
+	'Product marketing': i18n.translate( 'Product marketing' ),
+	'Product reviews': i18n.translate( 'Product reviews' ),
+	'Product sales': i18n.translate( 'Product sales' ),
+	'Professional cleaning service': i18n.translate( 'Professional cleaning service' ),
+	'Professional portfolio': i18n.translate( 'Professional portfolio' ),
+	'Professional profile': i18n.translate( 'Professional profile' ),
+	'Professional services': i18n.translate( 'Professional services' ),
+	'Promotional marketing': i18n.translate( 'Promotional marketing' ),
+	'Promotional products': i18n.translate( 'Promotional products' ),
+	Promotions: i18n.translate( 'Promotions' ),
+	Property: i18n.translate( 'Property' ),
+	Psychotherapy: i18n.translate( 'Psychotherapy' ),
+	Quiz: i18n.translate( 'Quiz' ),
+	Quotes: i18n.translate( 'Quotes' ),
+	Quran: i18n.translate( 'Quran' ),
+	RVs: i18n.translate( 'RVs' ),
+	Racism: i18n.translate( 'Racism' ),
+	'Real estate': i18n.translate( 'Real estate' ),
+	Recipes: i18n.translate( 'Recipes' ),
+	'Recreation center': i18n.translate( 'Recreation center' ),
+	Recruiting: i18n.translate( 'Recruiting' ),
+	Recruitment: i18n.translate( 'Recruitment' ),
+	'Recruitment consultant': i18n.translate( 'Recruitment consultant' ),
+	Relationships: i18n.translate( 'Relationships' ),
+	Religion: i18n.translate( 'Religion' ),
+	'Renewable energy': i18n.translate( 'Renewable energy' ),
+	Renovation: i18n.translate( 'Renovation' ),
+	Reporter: i18n.translate( 'Reporter' ),
+	Research: i18n.translate( 'Research' ),
+	Researcher: i18n.translate( 'Researcher' ),
+	Resort: i18n.translate( 'Resort' ),
+	Resources: i18n.translate( 'Resources' ),
+	Restaurant: i18n.translate( 'Restaurant' ),
+	Resume: i18n.translate( 'Resume' ),
+	Retail: i18n.translate( 'Retail' ),
+	Reviews: i18n.translate( 'Reviews' ),
+	Robotics: i18n.translate( 'Robotics' ),
+	Romance: i18n.translate( 'Romance' ),
+	Running: i18n.translate( 'Running' ),
+	Sailing: i18n.translate( 'Sailing' ),
+	Sales: i18n.translate( 'Sales' ),
+	Salon: i18n.translate( 'Salon' ),
+	'Saving money': i18n.translate( 'Saving money' ),
+	Savings: i18n.translate( 'Savings' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( '% savings', { context: 'A percentage discount, eg: 20% savings'} )` *ES Score: 16*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/translate-suggestions-about)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).